source/renderer/app/components/staking/stake-pools/StakePools.js

```diff
@@ -293,21 +293,43 @@ export default class StakePools extends Component<Props, State> {
                   </span>
                 </h2>
                 <StakingPageScrollContext.Consumer>
-                  {(stakePoolsScrollContext) => (
-                    <StakePoolsList
-                      stakePoolsList={stakePoolsDelegatingList}
-                      onOpenExternalLink={onOpenExternalLink}
-                      currentTheme={currentTheme}
-                      containerClassName="StakingWithNavigation_page"
-                      onSelect={this.onDelegate}
-                      numberOfRankedStakePools={numberOfRankedStakePools}
-                      isGridRewardsView={isGridRewardsView}
-                      showWithSelectButton
-                      scrollElementRef={
-                        stakePoolsScrollContext.scrollElementRef
-                      }
-                    />
-                  )}
+                  {(stakePoolsScrollContext) =>
```

Comment:
if the `stakePoolsList` component is the only one using `<StakingPageScrollContext.Consumer>` why not start off with the `isListView ? ` check and only include the consumer for `stakePoolsList` component

Reply:
Right, updated. Thanks!